src/Neo/Extensions/HashSetExtensions.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2015-2024 The Neo Project.
+//
+// HashSetExtensions.cs file belongs to the neo project and is free
+// software distributed under the MIT software license, see the
+// accompanying file LICENSE in the main directory of the
+// repository or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using Neo.IO.Caching;
+using System;
+using System.Collections.Generic;
+
+namespace Neo.Extensions
+{
+    /// <summary>
+    /// A helper class that provides common functions.
+    /// </summary>
+    public static class HashSetExtensions2
```

Comment:
extension2?

Reply:
yes we already have `HashSetExtensions` in the `neo.extensions.dll`

Reply:
Why not join them?

Reply:
`HashSetExtensions` doesnt require any dependencies inside `neo` library and `HashSetExtensions2` does. So can't be done

Reply:
We can use the same name with different namespaces, but the 2, it's weird

Reply:
If you insist, but you won't see this name and it shouldn't be used.